src/katgpucbf/dsim/__init__.py

```diff
@@ -1,7 +1,7 @@
 # noqa: D104
 
 ################################################################################
-# Copyright (c) 2021-2022, National Research Foundation (SARAO)
+# Copyright (c) 2021, National Research Foundation (SARAO)
```

Comment:
This change seems ... counterintuitive... is it just reverted to an older situation by removing lines 22-25?

Reply:
Yes. If you look at the diff against `main`, there are no changes to this file. I could squash this commit with [43dbf4f](https://github.com/ska-sa/katgpucbf/pull/426/commits/43dbf4f63f8b7b877a3bc48d6e1e30661004c0bc) so that there is no spurious history for this file.

Reply:
Nah, no need for that I think.